twoinputs/demand-windowsize.ipynb

```diff
@@ -0,0 +1,831 @@
+      "text/plain": [
+       "<Figure size 1152x648 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(16,9))\n",
+    "plt.suptitle(\"UIUC Demand Prediction with ESN\", fontsize=21)\n",
+    "plt.title(param_string(params))\n",
+    "plt.ylabel(\"UIUC Demand [kW]\", fontsize=16)\n",
+    "plt.xlabel(f\"Hours since {time_range[0]}\", fontsize=16)\n",
+    "# plot the truth\n",
+    "plt.plot(hour_range[-2*futureTotal:],norm*demand_norm[-2*futureTotal:],'b',label=\"True Demand\", \n",
+    "         alpha=0.7,\n",
+    "         color='tab:blue')\n",
+    "# # plot the prediction\n",
+    "plt.plot(hour_range[-futureTotal:], norm*init_pred.T[0],  alpha=0.8, \n",
```

Comment:
Consider adding spaces after colons.

Reply:
I didn't mark all of them, but all the plots are like this so it is probably fine.